### DISCUSSION TITLE
2023년 04월 15일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#37] [General]
# 2023년 04월 15일

LOG 월급을 받아서 토리, 망고, 포이랑 저녁에 표현 식당에 가서 고기를 먹었다. 표현 식당 처음 가봤는데 사장님이 우리 명찰을 보시고 예전에 회식 왔던 사람들이랑 같은 회사(?)냐고 물어보셔서 신기했다 ㅋㅋㅋ 2단계 구현을 완료했다. 구현할 게 …

https://cyma-s.github.io/2023-04-15/

**Comment by Choi-JJunho** [comment]
꾸준함이 중요한거죠~
베로 너무 잘하고 있어요 👍

**Reply by Cyma-s** [reply]
선플단 주노의 댓글 감사하다~

**Comment by be-student** [comment]
와 치킨이 1주일!

**Reply by Cyma-s** [reply]
결국 절반밖에 못 먹음...ㅋㅋㅋㅋㅋ
나머지는 쓰레기통으로
